Source/WebGPU/WebGPU/Buffer.mm

```diff
@@ -166,11 +166,6 @@ static MTLStorageMode storageMode(bool deviceHasUnifiedMemory, WGPUBufferUsageFl
 
 WTF_MAKE_TZONE_ALLOCATED_IMPL(Buffer);
 
-id<MTLBuffer> Buffer::makeIndexIndirectBuffer()
```

Comment:
this function wasn't used so I removed it since it had the ` + sizeof(uint32_t)`